src/wp-content/themes/twentynineteen/style-editor.scss

```
@@ -1020,3 +1020,13 @@ $group-block-background__padding: $font__size_base;
 		width: 100%;
 	}
 }
+
+/* Post Author Block  */
```

```suggestion
/** === Post Author Block === */
```

Use same formate for comment.